test/regridder_tests.jl

```diff
@@ -314,3 +315,72 @@ for FT in (Float32, Float64)
         end
     end
 end
+# test dataset truncation 
+@testset "test dataset truncation" begin
+    # Get the original dataset set up 
+    include(joinpath(pkgdir(ClimaCoupler), "artifacts", "artifact_funcs.jl"))
+    sst_data_all = joinpath(sst_dataset_path(), "sst.nc")
+    ds = NCDataset(sst_data_all, "r")
+    dates = ds["time"][:]
+    first_date = dates[1]
+    last_date = last(dates)
+
+    # set up comms_ctx 
+    device = ClimaComms.device()
+    comms_ctx = ClimaComms.context(device)
+    ClimaComms.init(comms_ctx)
+
+    # make path for truncated datasets 
+    COUPLER_OUTPUT_DIR = joinpath("experiments/AMIP/output", "tests")
+    mkpath(COUPLER_OUTPUT_DIR)
+
+    REGRID_DIR = joinpath(COUPLER_OUTPUT_DIR, "regrid_tmp/")
```

Comment:
Including the "/" in the path might cause problems on windows since it uses the backslash "\" instead. I'm not sure if this would be handled by joinpath though